README.md

```diff
@@ -19,15 +17,25 @@ To run the project:
         export DB_PASSWORD="communityfund_password"
 3. Run start
 
-        source .env && yarn start-dev-db
-        source .env && yarn start
+        yarn start:all
```

Comment:
What do we want to do with `.env`? Do we want the users to source it before hand or should we bake it into the commands?